tests/test_app/TestApp/Model/Enum/ArticleStatusLabel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @since         5.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace TestApp\Model\Enum;
+
+enum ArticleStatusLabel: string
+{
+    case PUBLISHED = 'Y';
+    case UNPUBLISHED = 'N';
+
+    public function label(): string
+    {
+        return 'Is ' . strtolower($this->name);
```

Comment:
mb_ as best practice maybe?

Reply:
This is a test class, not code that's going to be run for users so no need to be pedantic about it :)